scripts/uml-tests.sh

```diff
@@ -0,0 +1,10 @@
+sleep 10 # wait for jjs-frontend to start up
+
+cat > test.cpp << EOF
+#include <iostream>
+int main(){int64_t a,b;std::cin>>a>>b;std::cout<<a+b<<"\\n";}
+EOF
+
+jjs-cli submit --contest trial --filename test.cpp --problem A --toolchain g++
+
+{ sleep 1; poweroff; } &
```

Comment:
Why  `sleep` here?

Reply:
I don't remember exactly. Probably it's just to finish the init.d script before shutdown